docs/content/reference/static-configuration/cli-ref.md

```diff
@@ -354,14 +354,20 @@ Enable ConsulCatalog backend with default settings. (Default: ```false```)
 `--providers.consulcatalog.cache`:  
 Use local agent caching for catalog reads. (Default: ```false```)
 
+`--providers.consulcatalog.connectaware`:  
+Enable Consul Connect support. (Default: ```false```)
+
+`--providers.consulcatalog.connectbydefault`:  
+Automatically connect to a service via Consul connect. (Default: ```false```)
+
```

Comment:
```suggestion
```

As we have the `providers.consulcatalog.exposedByDefault` provider configuration option that tell traefik to (de)activate the auto discovery of services, and we have the `traefik.connect=true` tag that tell traefik which services are "connect" ready. It  is not necessary to have this `providers.consulcatalog.connectbydefault` flag.

Reply:
It is useful to have a secure cluster by default. With connectbydefault it will behave as if every service has `traefik.connect=true`. This is an extra protection if the operator forgets to set it (and if you want to go all in you do not need to put the extra label on every service). WDYT?

Reply:
> It is useful to have a secure cluster by default. With connectbydefault it will behave as if every service has traefik.connect=true. This is an extra protection if the operator forgets to set it (and if you want to go all in you do not need to put the extra label on every service). WDYT?

After thinking about it, it could be more user-friendly as it avoids writing connect tags for new services and can be a good thing to respect the more "secured" configuration.